draft-ietf-quic-multipath.md

```diff
@@ -240,12 +240,11 @@ ACK frames when used with the multipath extension acknowledge packets for the pa
 As multipath support is unknown during the handshake, acknowledgments of
 Initial and Handshake packets are sent using ACK frames.
 
-After the handshake concluded and both endpoints support the multipath extension,
+After the handshake concluded with support for the multipath extension,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
-including for acknowledging so far unacknowledged 0-RTT packets using Path ID 0.
-Endpoints MUST process ACK frames that acknowledge 0-RTT packets or 1-RTT packets,
-even when this extension is used. For example,
-ACK frames might be preferred by the sender as long as only the initial path
+also so far unacknowledged 0-RTT packets using Path ID 0.
+Endpoints MUST still process ACK frames that acknowledge 0-RTT packets or 1-RTT packets.
+For example, ACK frames might be preferred by the sender as long as only the initial path
 with Path ID 0 is in use.
```

Comment:
No strong opinion, but this will be interpreted as "please use ACK frames for path 0 if it is the only path." Maybe replace by "ACK frames might be preferred by some senders for various reasons."

Reply:
The intention here was to give an example. The sentence you proposed is a bit meaningless. If there is a real concern, we should just remove the sentence. But I think it's useful to give an example as this was the one case that was brought up. I don't think this is a risk of being see as recommendation as I think our recommendation is clear. At the end it will be an implementation decision potentially mostly related to implementation complexity. However, the important point here is that the other end needs to be able to handle this case.